app/src/main/java/no/usn/mob3000/domain/viewmodel/ProfileViewModel.kt

```diff
@@ -0,0 +1,37 @@
+package no.usn.mob3000.domain.viewmodel
+
+/**
+ * @author Hussein Abdul-Ameer
+ * created on : 01.11.2024
```

Comment:
Remember to use GitHub usernames and ISO8601 conformed dates. Additionally, this is an incorrectly placed top-level docstring, and this file does not need one that has special considerations. This should be removed. Instead, document classes, objects, functions and such as normal.